lib/cylc/gui/updater.py

```diff
@@ -221,9 +221,9 @@ def reconnect(self):
                     gobject.idle_add(self.warn, str(exc))
             return
 
-        self.app_window.set_title("%s - %s:%s" % (
-            self.cfg.suite,
-            self.suite_info_client.host,
+        gobject.idle_add(
+            self.app_window.set_title, "%s - %s:%s" % (
+            self.cfg.suite, self.suite_info_client.host,
```

Comment:
The `pep8` test is failing in my environment, wanting a bit more indentation here.